ShowBuildMenu.bat

```diff
@@ -15,7 +15,7 @@ echo.
 echo --- TESTING ---
 echo B. (Step 1) Set up a new test configuration for a particular database.
 echo C. (Step 2) Activate a test configuration.
-echo D. (Step 3) Run tests using active configuration.
+echo D. (Step 3) Run tests using active configuration (Build in Visual Studio).
```

Comment:
No, I mean to run this test step, you have to build in Visual Studio first.  Performing the build from the menu outputs to a different location.